File Formats/Images/UCG Image Format version 3.md

```diff
@@ -0,0 +1,291 @@
+# **Universal Compressed Graphics file format version 3**
+
+## Contents
+1. **Abstract**
+2. **Quick Overview**
+3. **Introduction & Purpose**
+4. **Specification** <br>
+	4.1 Definitions <br>
+	&emsp;4.1.1 Byte <br>
+	&emsp;4.1.2 Word <br>
+	&emsp;4.1.3 ColorValue <br>
+	&emsp;4.1.4 LengthValue <br>
+	&emsp;4.1.5 HuffmanTree <br>
+	&emsp;4.1.6 PixelData <br>
+5. **Explanation** <br>
+	5.1 Signature <br>
+	5.2 version <br>
+	5.3 Flags <br>
+	5.4 Width <br>
+	5.5 Height <br>
+	5.6 ColorHuffmanTree <br>
+	5.7 LengthHuffmanTree <br>
+	5.8 PixelData <br>
+	5.9 Huffman Codes <br>
+6. **See Also**
+
+## 1. Abstract
+This document defines and explains a file format for storing images and graphics, specificated for the usage with ComputerCraft and CraftOS 2.
+
+## 2. Quick Overview
+
+| Name       | Value             |
+| ---------- | ----------------- |
+| Type       | Image file format |
+| MIME       | ``image/ucg``     |
+| Extensions | ``.ucg``          |
```

Comment:
This should include a version key-value pair as well, as stated by https://github.com/oeed/CraftOS-Standards/blob/master/Standard%20Proposal%20Guidelines.md